src/pages/je-recrute-afpr-poei/inscription/index.page.test.tsx

```diff
@@ -173,6 +173,19 @@ describe('<JeRecruteAfprPoeiInscription />', () => {
 			expect(screen.getByText('Etape 3 sur 3')).toBeInTheDocument();
 		});
 
+		describe('lorsque l‘on rempli un commentaire', () => {
+			it('on ne peut pas remplir un text de plus de 500 caractères', async () => {
+				renderComponent();
+
+				await remplirFormulaireEtape1();
+				await directionNouvelleEtape();
+				await remplirFormulaireEtape2();
+				await directionNouvelleEtape();
+
+				const input = screen.getByRole('textbox', labelsEtape3[1]) as HTMLInputElement;
```

Comment:
J'ai un peu du mal à comprendre de quel champ il s'agit en ne lisant que le test.

Proposition, remplacer par : 
```suggestion
				const input = screen.getByLabelText('Commentaires ou autres informations utiles (500 caractères maximum)') as HTMLInputElement;
```

Reply:
Ou si on veut vraiment éviter une string qui n'est pas dans une variable, avoir éventuellement une variable type `LABEL_INPUT_COMMENTAIRE` réutilisée